src/assets/stack-blitz/src/main.ts

```diff
@@ -24,7 +25,9 @@ import {MaterialDocsExample} from './app/material-docs-example';
   entryComponents: [MaterialDocsExample],
   declarations: [MaterialDocsExample],
   bootstrap: [MaterialDocsExample],
-  providers: []
+  providers: [
+    { provide: MAT_FORM_FIELD_DEFAULT_OPTIONS, useValue: { appearance: 'fill' } },
```

Comment:
Can you please add a comment on why we do this?

Reply:
Added, PTAL.